docs/automated-process.md

```diff
@@ -71,13 +71,20 @@ Note that most of the following items cannot be done by yourself, but need someo
     The notification includes a link to the page where the upload step can be
     approved.
 
-2. Check out the PR in your `ansible-build-data` clone and copy the updated
-   porting guide from its `${MAJOR_VERSION}` directory into the
-   [`docs/docsite/rst/porting_guides/`](https://github.com/ansible/ansible-documentation/tree/devel/docs/docsite/rst/porting_guides/)
-   directory of the [`ansible-documentation`
-   repository](https://github.com/ansible/ansible-documentation/). Create a
-   PR for updating the porting guide for the `devel` branch of
-   `ansible-documentation`.
+2. To create the porting guide PR in the ansible-documentation repository, trigger [the automated
+   workflow](https://github.com/ansible/ansible-documentation/blob/devel/.github/workflows/release-porting-guide.yml)
+   on the **Actions** tab of the repository. This workflow has the following inputs,
```

Comment:
Why not also directly link to https://github.com/ansible/ansible-documentation/actions/workflows/release-porting-guide.yml?